docs/configuration/sessions.md

```diff
@@ -32,3 +33,26 @@ The following should be known when using this implementation:
 - Since multiple requests can be made concurrently to the OAuth2 Proxy, this session implementation
 cannot lock sessions and while updating and refreshing sessions, there can be conflicts which force
 users to re-authenticate
+
+
+### Redis Storage
+
+The Redis Storage backend stores sessions, encrypted, in redis. Instead sending all the information
```

Comment:
I think `of` got missing here
```suggestion
The Redis Storage backend stores sessions, encrypted, in redis. Instead of sending all the information
```